src/icrar/leap-accelerate/math/linear_math_helper.h

```diff
@@ -96,10 +99,11 @@ namespace icrar
 
         auto m = Eigen::Matrix<T, 3, 3>();
 
+        // column major to column major
```

Comment:
Same comment/question as above

Reply:
And while I'm here, why is there a special version for 3x3 matrices? The code looks exactly like `template<typename T, int R, int C> ConvertMatrix(const casacore::Matrix<T>& value)` so I'm not sure why this exists.

Reply:
good spot! will remove